### DISCUSSION TITLE
event-link for the list row

### DISCUSSION THREAD

**Discussion by teamtacho** [liedekef/events-made-easy#364] [General]
Hi
Can anyone guide me how to set the link to the event(page) for a entire row in the event-list?
Currently the link is only allocated to the field #_LINKEDNAME. I like to open the eventpage whereever I click on a field in the table
e.g. Date, Autorname, Seats ....

**Comment by liedekef** [comment]
Well, a couple of days ago I realized I didn't foresee a placeholder that easily returns the event url. The next version will have #_EVENTURL , so you can then use that to make anything clickable to your liking.

**Reply by teamtacho** [reply]
Ok. Great. Will this be used like <a href=#_EVENTURL>xxx</a>?

**Reply by teamtacho** [reply]
Sorry doesn't show up right "< a href=#_EVENTURL>" ...

**Reply by liedekef** [reply]
indeed, you'll be able to use it like that

**Reply by teamtacho** [reply]
Cool! Thanks.
